New package: TASOPT v3.0.0

JuliaRegistrator opened the pull request:
- Registering package: TASOPT
- Repository: https://github.com/MIT-LAE/TASOPT.jl
- Created by: @askprash
- Version: v3.0.0
- Commit: 3649ad0e04272da2372623073649a717d786364a
- Reviewed by: @askprash
- Reference: https://github.com/MIT-LAE/TASOPT.jl/commit/3649ad0e04272da2372623073649a717d786364a#commitcomment-155715797
- Description: Medium fidelity aircraft-propulsion system design and optimization. 
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd66fe0291d1882f6d211e0c42dcb2413ffb358e2e177a2692e57cd9315d2d2ae0dce510cc36db2c3808cff9fa72aea5fdb38fd5ec1789671911c8ecd08a52b0f10cc6aac98ed0d56b07249d5611d0c7ff40626c4ce482f9b1d11d7c39e5e7bedb022a936396e07d9eb5b707dbc9dce463eaf3b689424d1bc3569f06485392be6f76aef5c4177fd8a810a7652e170c9193a5d1a6abfc18207fff10cda53ab1a218d361d0950e49345f4ed538935786b0efe -->

askprash commented:
Hi, this is a package that is an evolution of a well known (in the aerospace community) FORTRAN code named TASOPT. I updated the package name to TASOpt.jl (which might actually be a better acronym) based on the automerge bot's complaint re all caps. But we would really not want to change the repo name since we already have many forks and many more clones. Can this please be merged without the repo name change? Alternatively I can go back to all caps name to match the current repo name. 

[noblock]

JuliaRegistrator committed:
New package: TASOPT v3.0.0

UUID: 22dd2482-b234-48ce-b849-20a4e029c823
Repo: https://github.com/MIT-LAE/TASOPT.jl.git
Tree: 957624ae4585d7a0670dc98ef25b5bfa1962be01

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
